test.py

```diff
@@ -228,6 +231,49 @@ def run_integration_tests(verbose=True):
     run_test_suite("tests/validation", "validation CLI tests", verbose=verbose)
 
 
+def clean_up_tests():
+    # remove files generated from running example files
```

Comment:
Can you add a bullet item in the "Create a new tutorial" section of https://github.com/NeurodataWithoutBorders/pynwb/blob/dev/docs/source/make_a_tutorial.rst to mention that any files generated by the tutorial should be add to this list.

Reply:
Done